maas/testing/fake_hp_monitoring.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+# Copyright 2015, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+args = sys.argv[1:]
+
+if args == ['-s', 'show server']:
+    print 'Status Ok'
+elif args == ['-s', 'show dimm']:
+    print 'Status Ok'
+elif args == ['ctrl', 'all', 'show', 'config']:
+    print 'logicaldrive OK)'
```

Comment:
I updated the commit message to specifically mention this. It's here so that the HP hardware plugin will run successfully and return metric data. So it's testing that the plugin works as originally written but obviously it's not testing the plugin works on a real system. This was added for use with rpc-heat. Hmmm, whether or not this is used should be configurable, that seems to have been lost from the code at some point. I'll add some tags.